xunit-autolabeler-v2/ast_parser/core/constants.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+IGNORED_METHOD_NAMES = (
+    'run_command',
+    'parse_command_line_args',
+    'main'
+)
+
+HTTP_CLASS_NAMES = (
+    'app',
+    'client'
+)
+
+HTTP_METHOD_NAMES = (
+    'get',
+    'post',
+    'put',
+    'patch',
+    'delete',
+    'options'
+)
```

Comment:
Not sure what this list is/will be used for for, but you are missing a few methods/verbs:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods

Reply:
Good catch - done.